docs/getting_started.md

```diff
@@ -0,0 +1,24 @@
+## Requirements:
+* Django 2.2
+* Wagtail 2.2
+* Python 3
+
+### To set up:
+ 1. Run `python3 pip install wagtail-content-import`.
+ 2. Add `'wagtail_content_import'` to `INSTALLED_APPS` above `wagtail.admin`
+ 3. Add the urls: include `wagtail_import.urls` in your `urlpatterns` in `urls.py`. This could look like:
+    
+        from django.conf.urls import include, url
```

Comment:
Format as ```python ?

Reply:
Unfortunately, this seems to be one of the quirks of mkdocs (https://www.mkdocs.org/user-guide/writing-your-docs/#writing-with-markdown, under "Fenced code blocks"): ```python blocks can't be indented, so I had to choose list formatting or syntax highlighting.

I thought it was useful having it as part of the numbered instructions, but happy to move it elsewhere and get the highlighting back if you think that would be better.

Reply:
Ah right, thanks for the explanation! Let's leave it like this.